crates/utils/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub type CacheLock<T> = std::sync::LazyLock<Cache<(), T>>;
 /// retries have already happened. Uses exponential backoff with maximum of one day. The first
 /// error is ignored.
 pub fn federate_retry_sleep_duration(retry_count: i32) -> Duration {
-  debug_assert!(retry_count != 0);
+  //debug_assert!(retry_count != 0);
```

Comment:
For some reason this assertion fails when calling `/api/v4/federated_instances` now.

Reply:
The problem is coming from [this call](https://github.com/LemmyNet/lemmy/blob/main/crates/db_views/readable_federation_state/src/impls.rs#L9). Somehow theres a row where last_retry_at is Some and also fail_count is 0 which is invalid for this function. But no idea why the ordering would affect this, maybe I just have something invalid in my local db.